src/webviews/extension-side/plotting/plotViewer.node.ts

```diff
@@ -36,7 +37,7 @@ export class PlotViewer extends WebviewPanelHost<IPlotViewerMapping> implements
         @inject(IThemeFinder) themeFinder: IThemeFinder,
         @inject(IWorkspaceService) workspaceService: IWorkspaceService,
         @inject(IApplicationShell) private applicationShell: IApplicationShell,
-        @inject(IFileSystem) private fs: IFileSystem
+        @inject(IFileSystem) private fs: FileSystem
```

Comment:
I think if there's a node specific IFileSystem, there should be a node specific interface. Passing the type here might make it compile, but inversify isn't going to enforce this. If we want it to be enforced in the container, we should have something like IFileSystemNode or inject the concrete type ```@inject(FileSystem)```

Reply:
@rchiodo I used the trick of using node specific `FileSystem` concrete types in *.node.ts` files to avoid the node specific fs APIs bleeding to the interface. 

Not exactly sure how inject work, are these two identical? 

```
@inject(IFileSystem) private fs: FileSystem
```

```
@inject(FileSystem) private fs: FileSystem
```

If they are identical, I will make the change and then introduce `IFileSystemNode` in next PR.

Reply:
Also I'm not sure which approach is the best yet, since we are doing all the refactoring and we will be an unhappy state for a while. I like the idea of `IFileSystemNode` but not sure if making that change would modify 50ish files and quite a few of them will need to inject both `IFileSystemNode` and `IFileSystem`, not a great code smell either.

Reply:
IFileSystemNode could be a superset of IFileSystem. The node files would inject the node specific one (well if they need it). It would be those you modified in this PR

And an object can implement more than one interface in the container. See examples of ```addBinding``` in some of the serviceRegistry files.

Reply:
If you're injecting FileSystem, then you need to register FileSystem as the thing in the container (not IFileSystem) or there will be binding errors on startup.

I think you could introduce IFileSystemNode without a lot of changes though.

Reply:
Looks like another spot where you missed putting it back